src/statespace/dart/WeldJoint.cpp

```diff
@@ -0,0 +1,32 @@
+#include <aikido/statespace/dart/WeldJoint.hpp>
+
+namespace aikido {
+namespace statespace {
+namespace dart {
+
+//=============================================================================
+WeldJoint::WeldJoint(
+      ::dart::dynamics::WeldJoint* _joint)
+  : Rn(_joint->getNumDofs())
+  , JointStateSpace(_joint)
+{
+}
+
+//=============================================================================
+void WeldJoint::convertPositionsToState(
+  const Eigen::VectorXd& _positions, StateSpace::State* _state) const
+{
+  setValue(static_cast<State*>(_state), _positions);
```

Comment:
I believe we don't need to convert any value here because the dimension of WeldJoint is zero.